visualization/app/codeCharta/state/settingsService/settings.service.events.ts

```diff
@@ -32,6 +32,14 @@ export interface SearchPatternSubscriber {
 	onSearchPatternChanged(searchPattern: string)
 }
 
+export interface FocusNodePathSubscriber {
```

Comment:
Let's rename this to `FocusNodeSubscriber` and `UnfocusNodeSubscriber`
and therefore `onFocusNodePath`  to `onFocusNode`
and `NODE_PATH_FOCUSED` to `NODE_FOCUSED_EVENT`